webapp/components/wpt-amend-metadata.js

```diff
@@ -164,6 +164,9 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
               <img class="browser" src="[[displayMetadataLogo(node.product)]]">
               :
               <paper-input label="Bug URL" value="{{node.url}}" autofocus></paper-input>
+              <template is="dom-if" if="[[!node.product]]">
+                <paper-input label="Label" value="{{node.label}}"></paper-input>
```

Comment:
In the screenshot there's a lowercase "label" here. Is that from something on this line, what does `{{node.label}}` mean? In any case, capitalizing it like "Label" just like "Bug URL" would be nice.

Reply:
Yea I already changed it. The lowercase label is from `label="label"` and `{{node.label}}` is the object that stores user input of `label`